### PR TITLE
hyperdrive: detail metrics

### PR DESCRIPTION
**Pull request by elithrar**
This PR details the metrics available in Hyperdrive.

### CODE REVIEW

File: content/hyperdrive/reference/metrics.md [hunk 1]
```diff
@@ -1,18 +1,46 @@
...
+| Metric                  | GraphQL Field Name         | Description                                                   |
+| ----------------------- | -------------------------- | ------------------------------------------------------------- |
+| Queries (qps)           | `count`                    | The number of queries issued against your Hyperdrive.         |
+| Cache Status            | `cacheStatus`              | Whether the query was cached or not. Can be one of `disabled`, `hit`, `miss`, or `uncacheable`. |
+| Query Bytes             | `queryBytes`               | The size of your queries, in bytes. |
+| Result Bytes            | `resultBytes`              | The size of your query _results_, in bytes. |
+| Connection Latency      | `connectionLatency`        | The time (in milliseconds) required to establish new connections from Hyperdrive to your database. |
+| Query Latency           | `queryLatency`             | The time (in milliseconds) required to query (and receive results) from your database. |
+| Query Latency           | `queryLatency`             | The time (in milliseconds) required to query (and receive results) from your database. |
```
Comment (mtlemilio) [inline content/hyperdrive/reference/metrics.md]: Duplicate line.

File: content/hyperdrive/reference/metrics.md [hunk 2]
```diff
@@ -1,18 +1,46 @@
...
-The metrics displayed in the Cloudflare dashboard charts are all pulled from Cloudflare's GraphQL Analytics API. You can access the metrics programmatically.
+## Metrics
 
-The Hyperdrive metrics are available in the `hyperdriveQueriesAdaptiveGroups` node under `viewer` > `accounts`. Refer to [Explore the GraphQL schema](/analytics/graphql-api/getting-started/explore-graphql-schema/) to learn how to navigate a GraphQL schema and discover which data are available.
+Hyperdrive currently exports the below metrics as part of the `hyperdriveQueriesAdaptiveGroups` GraphQL dataset:
 
-To learn more about the GraphQL Analytics API, refer to [GraphQL Analytics API](/analytics/graphql-api/).
+| Metric                  | GraphQL Field Name         | Description                                                   |
+| ----------------------- | -------------------------- | ------------------------------------------------------------- |
+| Queries (qps)           | `count`                    | The number of queries issued against your Hyperdrive.         |
```
Comment (mtlemilio) [inline content/hyperdrive/reference/metrics.md]: Not technically qps, it's the number of queries in whatever date time dimension, or total if no dimension specified.

File: content/hyperdrive/reference/metrics.md [hunk 3]
```diff
@@ -1,18 +1,46 @@
...
+| Cache Status            | `cacheStatus`              | Whether the query was cached or not. Can be one of `disabled`, `hit`, `miss`, or `uncacheable`. |
```
Comment (mtlemilio) [inline content/hyperdrive/reference/metrics.md]: There are a lot more possible values here. Would be nice to have an enum but that's not supported yet.

Comment (mtlemilio) [reply content/hyperdrive/reference/metrics.md]: Here are all the possible values for cacheStatus:
```
'disabled',
'hit',
'miss',
'multiplestatements',
'notaquery',
'oversizedquery',
'oversizedresult',
'parseerror',
'transaction',
'uncacheable',
'volatile'
```

File: content/hyperdrive/reference/metrics.md (hunk 2 above)
Comment (elithrar) [inline content/hyperdrive/reference/metrics.md]:
```suggestion
| Queries                 | `count`                    | The number of queries issued against your Hyperdrive in the given time period.         |
```

File: content/hyperdrive/reference/metrics.md (hunk 1 above)
Comment (elithrar) [inline content/hyperdrive/reference/metrics.md]:
```suggestion
```

File: content/hyperdrive/reference/metrics.md (hunk 3 above)
Comment (elithrar) [inline content/hyperdrive/reference/metrics.md]:
```suggestion
| Cache Status            | `cacheStatus`              | Whether the query was cached or not. Can be one of `disabled`, `hit`, `miss`, `uncacheable`, `multiplestatements`, `notaquery`, `oversizedquery`, `oversizedresult`, `parseerror`, `transaction`, and `volatile`.
```

**Review by mtlemilio — APPROVED** [review]

**Review by Maddy-Cloudflare — APPROVED** [review]

**Pull request merged**